compliance_checker/cf/cf_1_6.py

```diff
@@ -26,8 +26,7 @@ class CF1_6Check(CFNCCheck):
     """CF-1.6-specific implementation of CFBaseCheck; supports checking
     netCDF datasets.
     These checks are translated documents:
```

Comment:
The Links below are broken. I replaced one with the CF docs but the other one seems to be gone.

![Screenshot from 2023-05-17 17-00-10](https://github.com/ioos/compliance-checker/assets/950575/7bc1f0c5-0cfc-4f01-81c0-6c0f3621b4b0)